### PR TITLE
Print `run-id` when starting a new job run.

### PR DESCRIPTION
**Pull request by rgo3**

### CODE REVIEW

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run mac integration tests

File: pkg/cmd/job/job_run.go [hunk 1]
```diff
@@ -17,7 +19,8 @@ func newCmdJobRun(ctx api.Context) *cobra.Command {
 				return err
 			}
 
-			_, err = client.RunJob(args[0])
+			run, err := client.RunJob(args[0])
+			fmt.Fprintln(ctx.Out(), run.ID)
```
Comment (bamarni) [inline pkg/cmd/job/job_run.go:26]: we should check for error here and return early, as "run" might be nil.

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Pull request merged**